test/testsuites/packagestests.go

```diff
@@ -82,3 +89,33 @@ func childResourcesNotCreated(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// testOpSrcWithConflictingPackages ensures that an OperatorSource and its child resources
+// are successfully rolled out regardless of whether or not there is a
+// package conflict with an exiting OperatorSource.
+func testOpSrcWithConflictingPackages(t *testing.T) {
+	ctx := test.NewTestCtx(t)
+	defer ctx.Cleanup()
+
+	// Get global framework variables.
+	client := test.Global.Client
+
+	// Get test namespace.
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		t.Errorf("Could not get namespace: %v", err)
+	}
+
+	// Create the OperatorSource.
+	opSrcName := "conflicting-operators"
+	err = helpers.CreateRuntimeObject(test.Global.Client, ctx, helpers.CreateOperatorSourceDefinition(opSrcName, namespace))
+	if err != nil {
```

Comment:
@awgreene Please use `assert`/`require` here and everywhere else you're checking for errors. 

Otherwise looks good to me.

Reply:
Will do!

Reply:
I am going to wait until your changes in #177 go through before implementing these changes.